test/unittests/state_transition_block_test.cpp

```diff
@@ -13,6 +13,7 @@ TEST_F(state_transition, block_apply_withdrawal)
     static constexpr auto withdrawal_address = 0x8888_address;
 
     block.withdrawals = {{0, 0, withdrawal_address, 3}};
+    block.timestamp = 1;
```

Comment:
I think you can put this as a default to `state_transition.block` definition in state_transition.cpp